src/mscorlib/shared/System/TimeZoneInfo.Win32.cs

```diff
@@ -37,9 +37,21 @@ private sealed partial class CachedData
             private static TimeZoneInfo GetCurrentOneYearLocal()
             {
                 // load the data from the OS
-                Win32Native.TimeZoneInformation timeZoneInformation;
-                long result = UnsafeNativeMethods.GetTimeZoneInformation(out timeZoneInformation);
-                return result == Win32Native.TIME_ZONE_ID_INVALID ?
+                long result;
+                Interop.Kernel32.TIME_ZONE_INFORMATION timeZoneInformation;
+
+#if CORERT
+                Interop.Kernel32.TIME_DYNAMIC_ZONE_INFORMATION dynamicTimeZoneInformation;
+                result = Interop.Kernel32.GetDynamicTimeZoneInformation(out dynamicTimeZoneInformation);
```

Comment:
@tarekgh corert calls GetDynamicTimeZoneInformation here, coreclr calls GetTimeZoneInformation. Is that intentional?